docs/process/release.md

```diff
@@ -67,38 +67,36 @@ In general, cherry picks for LTS should only be done if it meets one of the crit
 
 # Release
 
-Releasing is performed using Angular's unified release tooling. Each week, two releases are expected, `latest` and `next` on npm.
-
-**For a minor OR major release:**
-
-After FW releases `-rc.0` for an upcoming minor/major version, update the corresponding version in:
+Releasing is performed using Angular's unified release tooling. Each week, two releases are expected, `latest` and `next` on npm. For major
+and minor releases, some dependencies need to be manually bumped. The following files contain all the version numbers which need to be
+manually updated:
 
 - [`latest-versions.ts`](/packages/schematics/angular/utility/latest-versions.ts#L=18)
 - [`latest-versions/package.json`](/packages/schematics/angular/utility/latest-versions/package.json)
 - [`@angular/pwa`](/packages/angular/pwa/package.json)
 - [`@angular-devkit/build-angular`](/packages/angular_devkit/build_angular/package.json)
 - [`@ngtools/webpack`](/packages/ngtools/webpack/package.json)
 
-The same needs to be done for a `-next.0` release, and needs to be done for both minor _and_ major
-releases.
+**BEFORE a minor OR major CLI release:**
+
+After FW releases `-rc.0` for an upcoming minor/major version, update the corresponding version in the above files to match `-rc.0`.
+
```

Comment:
This is actually not correct. We should not need to update to `-RC.0`. If we want to do so, we'd need to update the release tool checks.

Reply:
Right, I remember this was a challenge. I would like to find a path to do this just so users' `package.json` files reflect the RC state, but we can follow up separately on that. Removed this line for now.